feat(l10n): Add Simplified Chinese localization

Juitar opened the pull request:
Update and complete Simplified Chinese localization

- Merged translations from previous versions
- Added missing translations for new items

Juitar committed:
feat(l10n): Add Simplified Chinese localization

Juitar committed:
put the Chinese locazation file in right lang file folder

KreloX commented:
The Myrmex trait is renamed to Non-arthropod Damage Bonus in newer versions, but not in the Chinese lang file, this should probably be made consistent. Also a similar issue with some of the traits' descriptions, Non-arthropod Damage Bonus should specify the amount and Poisoned and Iced the duration of the effect in seconds.

Juitar committed:
Update the chinese locazation file about Ice and non-arthropod_damage_bonus

Juitar commented:
I've made the following updates to the Chinese localization file based on your feedback:
-Added duration details to the `Poisoned` and `Iced` trait descriptions
-Added specific bonus values to the `Non-arthropod Damage Bonus` description
-Renamed the `Myrmex` trait to `非节肢生物伤害加成` (Non-arthropod Damage Bonus) for consistency.
